MKOB/MKOB.pyw

```diff
@@ -24,19 +24,25 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
 
-"""MKOB.pyw
```

Comment:
I know it doesn't look great - but following the `"""` immediately by the module name is used to generate documentation (at the point we decide to do that). If there is a line break, the doc processing won't pick up the module name for the documentation.

Reply:
> I know it doesn't look great - but following the `"""` immediately by the module name is used to generate documentation (at the point we decide to do that). If there is a line break, the doc processing won't pick up the module name for the documentation.

Sorry about that. I can fix it.

When I read the `docstring` documentation that you pointed me to, it claimed that it works equally well if the module name is on the next line, but I never did try it to see what happens.

Reply:
Ah, okay... Maybe it has changed since I used it. Leave it as is (if you haven't changed it) and we can try it.